arangod/Aql/AqlItemBlock.cpp

```diff
@@ -72,7 +81,8 @@ inline void CopyValueOver(std::unordered_set<AqlValue>& cache, AqlValue const& a
 
 /// @brief create the block
 AqlItemBlock::AqlItemBlock(AqlItemBlockManager& manager, size_t nrItems, RegisterCount nrRegs)
-    : _nrItems(nrItems), _nrRegs(nrRegs), _manager(manager), _refCount(0), _rowIndex(0) {
+    : _valueCount(1, AqlValue::SimpleHash(), AqlValue::SimpleEqual()),
```

Comment:
Maybe, I don't know. I used 1 because that seems to be value that stdlib uses for an empty map with g++9.3.
I copied that value just because I couldn't figure out what would be the ideal value here.
Maybe we should do a bit more research here...

Reply:
Just verified that constructing it with a bucket_count value of 1 does one allocation more than constructing it with a bucket_count value of 0. At least with the stdlib shipped with g++9.3.
But there it makes sense to use a bucket_count value of 0, indeed.

Reply:
Now constructing the map with a bucket_count of 0